docs/core/UPGRADE_NEW_VERSION.md

```diff
@@ -0,0 +1,71 @@
+# How to upgrade to a new version of Otter
+
+We recommend using `ng update` command from @angular/cli to perform the update:
+
+```shell
+# Upgrade to latest
+yarn ng update @o3r/core
+
+# Upgrade to a specific version
+yarn ng update @o3r/core@10.0.0-rc.1
+
+# Upgrade to a specific range
+yarn ng update @o3r/core@^10
+```
+
+Using this command will trigger the update for @o3r packages, @ama-sdk packages and all related dependencies. 
+
+Using `ng update` will execute the migration scripts provided by the packages on top of updating the versions in your `package.json`.
+
+This can greatly reduce the effort needed to migrate as some of the updates on your code can be performed automatically.
+
+> [!CAUTION]
```

Comment:
Maybe more an `!importante` according to the github discussion https://github.com/orgs/community/discussions/16925 

(maybe the same for the following)

Reply:
grazie mille